addons/mail/static/src/scss/mail_activity.scss

```diff
@@ -33,7 +33,7 @@
 .o_list_view, .o_legacy_list_view {
     .o_list_table tbody > tr {
         > td.o_data_cell.o_list_activity_cell {
-            overflow: visible !important; // allow the activity dropdown to overflow
+            overflow: visible; // allow the activity dropdown to overflow
```

Comment:
Why not remove all the rule ?
Where this rule is used as now we have portal to avoid stacking context.